src/client.ts

```diff
@@ -135,14 +136,21 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
 
     // Telemetry for specific events
     // onPendingSession, onSession, onMediaStarted, onSessionTerminated logged in event handlers
-    this.on('sdkError', this.logger.error.bind(this.logger));
+    this.on('sdkError', (error: SdkError) => {
+      /* logging errors is dangerous because they could contain PII */
+      this.logger.error(error, null, true);
```

Comment:
I don't like blanket sweeping all errors like this. The only issue is we want these errors to contain useful information, but sometimes that information is considered _Customer Data_. If we limit what goes into the error, we are limiting the usefulness of them. This seemed like the better option to just skip the server than trying to track down all the errors thrown.

Reply:
hmm. we can probably roll with this for now. I have mixed feelings here.